src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
-
 // write your code here
+
+const populationElement = [...document.querySelectorAll('.population')];
+const totalPopulationElement = document.querySelector('.total-population');
+const averegePopulationElement = document.querySelector('.average-population');
```

Comment:
There is a spelling mistake in the variable name 'averegePopulationElement'. It should be 'averagePopulationElement' to match the class name in the HTML.